### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by pmac**
* Ensure docker environment is setup for l10n linting (use make)
* Ensure docker build args are passed to docker (GIT_SHA, and LOCALE_ENV)

### CODE REVIEW

**Comment by pmac** [comment]
@jpetto I discovered that the new l10n automation wasn't working when a `.env` file wasn't present for docker-compose ¯\_(ツ)_/¯. So I've fixed that by using make which ensures that file is created. Also it turns out that the `GIT_SHA` and `LOCALE_ENV` variables weren't making it to the docker images and were therefore not working. This fixes that as well.

File: docker-compose.yml [hunk 1]
```diff
@@ -59,36 +60,52 @@ services:
...
     image: mozmeao/kitsune:locales-${GIT_COMMIT_SHORT:-latest}
 
   full-no-locales:
     build:
       context: .
       target: full-no-locales
+      args:
+        - GIT_SHA
+        - LOCALE_ENV
+    image: mozmeao/kitsune:locales-${GIT_COMMIT_SHORT:-latest}
```
Comment (jpetto) [inline docker-compose.yml]: Two images? Is this copy pasta?

Comment (pmac) [reply docker-compose.yml]: Indeed it is! Good catch!

**Comment by pmac** [comment]
Fixed. Thanks for the quick r? 🎆

**Pull request merged**